### PR TITLE
Remove parseFile from toml

### PR DESCRIPTION
**Pull request by jcao219**
Feel free to close if this is not the right direction

### CODE REVIEW

File: toml/parser.ts [hunk 1]
```diff
@@ -539,10 +539,18 @@ export function parse(tomlString: string): object {
...
   return parse(strFile);
 }
+
+export async function parseFile(filePath: string): Promise<object> {
+  if (!(await exists(filePath))) {
+    throw new Error("File not found");
+  }
+  const strFile = await readFileStr(filePath);
+  return parse(strFile);
+}
```
Comment (ry) [inline toml/parser.ts]: Actually I'd rather that we remove the `parseFileSync` and leave it up to the caller to load the data. The TOML module should only deal with parsing, not with I/O.

Would you mind a revised PR that removes `parseFileSync` (or just moves it to the test module - if that is needed there)?

File: toml/parser.ts [hunk 2]
```diff
@@ -275,23 +275,23 @@ class Parser {
...
           }
           break;
         case `"`:
-          if (inLitteral) {
-            inLitteral = false;
+          if (inLiteral) {
+            inLiteral = false;
           } else {
-            inLitteral = true;
+            inLiteral = true;
```
Comment (ry) [inline toml/parser.ts]: Ah - thanks!

File: toml/parser.ts (hunk 1 above)
Comment (jcao219) [reply toml/parser.ts]: That makes a lot of sense to me!

**Comment by jcao219** [comment]
Done

**Pull request merged**